locales/en/yourloops.json

```
@@ -141,10 +148,11 @@
   "events": "Events",
   "events-configuration": "Events configuration",
   "events-list": "Events list",
+  "feedback-checkbox": "Feedback checkbox",
   "female": "Female",
   "field-with-exactly-15-characters": "This field must be exactly 15 characters long",
   "file-uploaded-pdf-jpeg-png": "File uploaded (PDF, JPEG, PNG)",
-  "firstname": "First Name",
+  "firstName": "First Name",
```

```suggestion
  "first-name": "First name"
```
I don't think that 'name' should start with an uppercase, this should be changed in the components that it is used.